package.json

```diff
@@ -33,6 +33,7 @@
 		"build": "tsc --project tsconfig.build.json",
 		"build:watch": "npm run build -- --watch --incremental",
 		"lint": "eslint .",
+		"lint:required": "eslint --no-eslintrc -c .eslintrc.required.json .",
```

Comment:
Can you add some more info on this lint changes? Do we need to make different updates for open source side of things?

Reply:
> Can you add some more info on this lint changes? Do we need to make different updates for open source side of things?

Here in the PR or in the docs?

Can start just like it is in the closed repo, but it does need updating to the newer eslint 9.x config format (which I'm working on locally).